spec/classes/init_spec.rb

```diff
@@ -702,4 +761,63 @@
       end
     end
   end
+
+  describe 'with enable_driftfile set to' do
```

Comment:
Yes, I agree, that would be better. I could also reuse $driftfile, if it's empty I disable the complete block. This way we don't need an additional variable. I'll give it a try, let's see if it looks good.